src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Optimize Cumulative Layout Shift
+subhead: Learn how to avoid sudden layout shifts to improve user-experience
+authors:
+  - addyosmani
+date: 2020-05-05
+hero: hero.png
+description: |
+  Cumulative Layout Shift (CLS) is a metric that quantifies how often users experience sudden shifts in page content. In this guide, we'll cover optimizing common causes of CLS such as images and iframes without dimensions or dynamic content.
+alt: Layout shifts can suddenly push the content you are reading or are about to click further down the page, leading to a poor user-experience. Reserving space for dynamic content causing layout shifts leads to a more delightful user experience.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - fast
+  - web-vitals
+  - cumulative-layout-shift
+  - cls
+  - lighthouse
+---
+
+"I was about to click that! Why did it move? 😭"
+
+Layout shifts can be distracting to users. Imagine you've started reading an article when all of a sudden elements shift around the page, throwing you off and requiring you to find your place again. This is very common on the web, including when reading the news, or trying to click those 'Search' or 'Add to Cart' buttons. Such layout shifts happen when elements on a page move because another element was suddenly added to the page or resized. 
+
+[Cumulative Layout Shift](/cls) (CLS) - a [Core Web Vitals](/metrics) metric, measures the instability of content by summing shift scores across layout shifts that don't occur within 500ms of user input. It looks at how much the viewport shifted as well as the distance the element impacted was shifted.
```

Comment:
Also, can you link to `/vitals/`, which should be live before you merge this PR?

Reply:
Certainly. Updating now.